README.md

```diff
@@ -2,6 +2,11 @@
 
 ![alt text](readme/intellijIdea_RunnerConfig.png "Intellij idea configuration")
 
+##Creating Constraints
```

Comment:
El contraint se ejecuta despues o antes de tener datos? 

Lo digo porque neo4j creo los nodos en la carga, no se si lance error lanzar el contraint antes de tener los datos... ??

Reply:
El constraint se crea antes de tener datos, por eso te pregunte donde estaban los comandos de carga de información.

Reply:
Ohhh... nice to know that :D